src/presentation/test/integration/service-availability.test.ts

```diff
@@ -0,0 +1,43 @@
+import { expect, jest, test } from '@jest/globals';
+import request from "supertest";
+import app from "./app";
+import { WHICH_TYPE_URL } from "../../controller/registration/Routing";
+import * as configConstants from "../../../config/constants";
+import enTranslationText from "../../../../locales/en/translations.json";
+import { HEALTHCHECK_URL } from '../../controller/global/Routing';
+
+const SERVICE_UNAVAILABLE_TEXT = "Sorry, the service is unavailable";
+
+describe("Service Availability tests", () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  const setServiceUnavailable = (value: boolean) => {
+    jest.replaceProperty(configConstants, 'SHOW_SERVICE_UNAVAILABLE_PAGE', value);
+  };
```

Comment:
was unable to get it working with replace process.env, the replace worked but config had already got the process env values before we can change them, so had to replace the property in config.